MainApp/templates/game_template.html

```diff
@@ -0,0 +1,4 @@
+<div class="col-4">
+    {% load static %}
```

Comment:
template tags should be on the first line of the file. Move `{% load static %}` to the top please